CHANGES_NEXT_RELEASE

```diff
@@ -3,4 +3,5 @@
 - [cygnus-common][SQL] Fix expiration records select with a limit to avoid java out of memory error (#2273)
 - [cygnus-ngsi] Removes "_" in schema name for DM -schema family (#2270, #2201 reopens for Postgis)
 - [cygnus-ngsi] UPGRADE: Debian version from 11.6 to 12.1 in Dockerfile
+- [cygnus-ngsi] UPGRADE: Java 11 to Java 17 used in Dockerfile
```

Comment:
Java 17 should be enabled in ci.yml (currently, the line is commented)

Reply:
Java 16 and Java 17 tests are failing due to some tests has problems with class instantation of :
mockito 
java.regex

Reply:
https://github.com/telefonicaid/fiware-cygnus/issues/2297